openapi/core-data.yaml

```diff
@@ -438,7 +434,6 @@ components:
             - deviceName: "device-001"
               resourceName: "resource-001"
               profileName: "profile-001"
-              id: "31569347-9369-43ec-aa6a-59ea9c624a6f"
```

Comment:
If users choose to use redis client, the response will still include reading id, right?

Reply:
Right, since we use Postgres as the primary DB, we can consider removing it from the Swagger doc.